2019-04-01T21:05:40.668+0000 I CONTROL  [initandlisten] MongoDB starting : pid=3489 port=27017 dbpath=/data/db 64-bit host=ip-172-31-18-151
2019-04-01T21:05:40.669+0000 I CONTROL  [initandlisten] db version v3.6.3
2019-04-01T21:05:40.669+0000 I CONTROL  [initandlisten] git version: 9586e557d54ef70f9ca4b43c26892cd55257e1a5
2019-04-01T21:05:40.669+0000 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.1.0g  2 Nov 2017
2019-04-01T21:05:40.669+0000 I CONTROL  [initandlisten] allocator: tcmalloc
2019-04-01T21:05:40.669+0000 I CONTROL  [initandlisten] modules: none
2019-04-01T21:05:40.669+0000 I CONTROL  [initandlisten] build environment:
2019-04-01T21:05:40.669+0000 I CONTROL  [initandlisten]     distarch: x86_64
2019-04-01T21:05:40.669+0000 I CONTROL  [initandlisten]     target_arch: x86_64
2019-04-01T21:05:40.669+0000 I CONTROL  [initandlisten] options: { processManagement: { fork: true }, systemLog: { destination: "file", path: "/home/ubuntu/databaseTest/log/mongodb.log" } }
2019-04-01T21:05:40.669+0000 I -        [initandlisten] Detected data files in /data/db created by the 'wiredTiger' storage engine, so setting the active storage engine to 'wiredTiger'.
2019-04-01T21:05:40.669+0000 I STORAGE  [initandlisten] 
2019-04-01T21:05:40.669+0000 I STORAGE  [initandlisten] ** WARNING: Using the XFS filesystem is strongly recommended with the WiredTiger storage engine
2019-04-01T21:05:40.669+0000 I STORAGE  [initandlisten] **          See http://dochub.mongodb.org/core/prodnotes-filesystem
2019-04-01T21:05:40.669+0000 I STORAGE  [initandlisten] wiredtiger_open config: create,cache_size=256M,session_max=20000,eviction=(threads_min=4,threads_max=4),config_base=false,statistics=(fast),log=(enabled=true,archive=true,path=journal,compressor=snappy),file_manager=(close_idle_time=100000),statistics_log=(wait=0),verbose=(recovery_progress),
2019-04-01T21:05:41.479+0000 I STORAGE  [initandlisten] WiredTiger message [1554152741:479370][3489:0x7f3ba19f6f80], txn-recover: Main recovery loop: starting at 6/16640
2019-04-01T21:05:41.565+0000 I STORAGE  [initandlisten] WiredTiger message [1554152741:565916][3489:0x7f3ba19f6f80], txn-recover: Recovering log 6 through 7
2019-04-01T21:05:41.616+0000 I STORAGE  [initandlisten] WiredTiger message [1554152741:616819][3489:0x7f3ba19f6f80], txn-recover: Recovering log 7 through 7
2019-04-01T21:05:41.746+0000 I CONTROL  [initandlisten] 
2019-04-01T21:05:41.746+0000 I CONTROL  [initandlisten] ** WARNING: Access control is not enabled for the database.
2019-04-01T21:05:41.746+0000 I CONTROL  [initandlisten] **          Read and write access to data and configuration is unrestricted.
2019-04-01T21:05:41.746+0000 I CONTROL  [initandlisten] 
2019-04-01T21:05:41.746+0000 I CONTROL  [initandlisten] ** WARNING: This server is bound to localhost.
2019-04-01T21:05:41.746+0000 I CONTROL  [initandlisten] **          Remote systems will be unable to connect to this server. 
2019-04-01T21:05:41.746+0000 I CONTROL  [initandlisten] **          Start the server with --bind_ip <address> to specify which IP 
2019-04-01T21:05:41.746+0000 I CONTROL  [initandlisten] **          addresses it should serve responses from, or with --bind_ip_all to
2019-04-01T21:05:41.746+0000 I CONTROL  [initandlisten] **          bind to all interfaces. If this behavior is desired, start the
2019-04-01T21:05:41.746+0000 I CONTROL  [initandlisten] **          server with --bind_ip 127.0.0.1 to disable this warning.
2019-04-01T21:05:41.746+0000 I CONTROL  [initandlisten] 
2019-04-01T21:05:41.758+0000 I FTDC     [initandlisten] Initializing full-time diagnostic data capture with directory '/data/db/diagnostic.data'
2019-04-01T21:05:41.759+0000 I NETWORK  [initandlisten] waiting for connections on port 27017
2019-04-01T21:16:33.393+0000 I NETWORK  [listener] connection accepted from 127.0.0.1:41256 #1 (1 connection now open)
2019-04-01T21:16:33.398+0000 I NETWORK  [conn1] received client metadata from 127.0.0.1:41256 conn: { driver: { name: "nodejs", version: "3.2.2" }, os: { type: "Linux", name: "linux", architecture: "x64", version: "4.15.0-1034-aws" }, platform: "Node.js v11.12.0, LE, mongodb-core: 3.2.2" }
2019-04-01T21:16:33.420+0000 I NETWORK  [conn1] end connection 127.0.0.1:41256 (0 connections now open)
2019-04-01T21:17:33.267+0000 I NETWORK  [listener] connection accepted from 127.0.0.1:41260 #2 (1 connection now open)
2019-04-01T21:17:33.271+0000 I NETWORK  [conn2] received client metadata from 127.0.0.1:41260 conn: { driver: { name: "nodejs", version: "3.2.2" }, os: { type: "Linux", name: "linux", architecture: "x64", version: "4.15.0-1034-aws" }, platform: "Node.js v11.12.0, LE, mongodb-core: 3.2.2" }
2019-04-01T21:17:33.289+0000 I STORAGE  [conn2] createCollection: mydb.CUSites with generated UUID: e57f046b-725f-4aab-beb0-7d271fe7edcf
2019-04-01T21:17:33.338+0000 I NETWORK  [conn2] end connection 127.0.0.1:41260 (0 connections now open)
2019-04-01T21:30:37.621+0000 I CONTROL  [signalProcessingThread] got signal 15 (Terminated), will terminate after current cmd ends
2019-04-01T21:30:37.621+0000 I NETWORK  [signalProcessingThread] shutdown: going to close listening sockets...
2019-04-01T21:30:37.621+0000 I NETWORK  [signalProcessingThread] removing socket file: /tmp/mongodb-27017.sock
2019-04-01T21:30:37.621+0000 I FTDC     [signalProcessingThread] Shutting down full-time diagnostic data capture
2019-04-01T21:30:37.623+0000 I STORAGE  [signalProcessingThread] WiredTigerKVEngine shutting down
2019-04-01T21:30:37.687+0000 I STORAGE  [signalProcessingThread] shutdown: removing fs lock...
2019-04-01T21:30:37.687+0000 I CONTROL  [signalProcessingThread] now exiting
2019-04-01T21:30:37.687+0000 I CONTROL  [signalProcessingThread] shutting down with code:0
